tests/TestCase/Database/Schema/TableTest.php

```diff
@@ -663,6 +663,48 @@ public function testTemporary()
     }
 
     /**
+     * Tests the toArray() method.
+     *
+     * @return void
+     */
+    public function testToArray()
+    {
+        $table = new Table('articles');
+        $result = $table->toArray();
+        $expected = [
+            'columns' => [],
+            'indexes' => [],
+            'constraints' => [],
+            'options' => [],
+            'typeMap' => [],
+            'temporary' => false,
+        ];
+        $this->assertSame($expected, $result);
+    }
+
+    /**
+     * Tests the __debugInfo() method.
+     *
+     * @return void
+     */
+    public function testDebugInfo()
+    {
+        $table = new Table('articles');
+        $table->setTemporary(true);
+
+        $result = $table->__debugInfo();
+        $expected = [
+            'columns' => [],
+            'indexes' => [],
+            'constraints' => [],
+            'options' => [],
+            'typeMap' => [],
+            'temporary' => true,
+        ];
+        $this->assertSame($expected, $result);
```

Comment:
Might be worth having a test for a table with columns.

Reply:
Since this just maps the internal property I thought checking the keys suffices, the actual table and columns are tested with the designated setters and getters I would think.